docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -0,0 +1,185 @@
+
+# MongoDB Atlas
+
+In this guide you will:
+
+1. Configure Teleport for accessing your MongoDB Atlas cluster.
+2. Configure self-managed X.509 authentication on your Atlas cluster.
+3. Connect to your Atlas cluster via Teleport.
+
+## Prerequisites
+
+- Teleport version `7.0` or newer.
+- [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) cluster.
+
+## Step 1/3. Configure Teleport
+
+### Setup Teleport Auth and Proxy services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Setup Teleport Database service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Start Teleport Database service:
+
+```bash
+teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=mongodb-atlas \
+  --protocol=mongodb \
+  --uri=mongodb+srv://cluster0.abcde.mongodb.net \
+  --ca-cert=/path/to/letsencrypt/root-ca
+  --labels=env=dev
+```
+
+See below for details on `--uri` and `--ca-cert` flags.
+
+<Admonition type="note">
+  The `--auth-server` flag must point to the Teleport cluster's proxy endpoint
+  because database service always connects back to the cluster over a reverse
+  tunnel.
+</Admonition>
+
+<Admonition type="tip">
+  You can start Database service using configuration file instead of CLI flags.
+  See [YAML reference](../reference/configuration.mdx).
+</Admonition>
+
+#### Connection endpoint (`--uri`)
+
+To find out your Atlas cluster's connection endpoint for the URI, use the
+Connect dialog on the Database Deployments overview page:
+
+![Connect](../../../img/database-access/guides/atlas/atlas-connect-btn@2x.png)
+
+Go through the "Setup connection security" step and select "Connect with the
+MongoDB shell" to view the connection string:
+
+![Connection string](../../../img/database-access/guides/atlas/atlas-connect@2x.png)
+
+Use only the scheme and hostname parts of the connection string in the URI:
+
+```txt
+--uri=mongodb+srv://cluster0.abcde.mongodb.net
+```
+
+#### Atlas CA certificate (`--ca-cert`)
+
+MongoDB Atlas uses certificates signed by Let's Encrypt as described in
+[Which certificate authority signs MongoDB Atlas cluster TLS certificates?](https://docs.atlas.mongodb.com/reference/faq/security/#which-certificate-authority-signs-mongodb-atlas-cluster-tls-certificates-)
+
+Download Let's Encrypt [root certificate](https://letsencrypt.org/certs/isrgrootx1.pem.txt)
+and use it as a CA in the database service configuration:
+
+```txt
+--ca-cert=/tmp/isrgrootx1.pem
+```
+
+### Create Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 2/3. Configure Atlas
+
+### Enable self-managed X.509 authentication
+
+Teleport will authenticate with MongoDB Atlas using
+[self-managed X.509 authentication](https://docs.atlas.mongodb.com/security-self-managed-x509/).
+
+First, obtain Teleport CA certificate by running the following `tctl auth sign`
+command against your Teleport cluster:
+
+```bash
+tctl auth sign --format=mongodb --host=mongo --out=mongo
```

Comment:
This works, but it's unclear with the `-ttl` is ?

Reply:
@benarent TTL flag isn't needed for this command. For Atlas we're only using CA certificate, database cert/key which you configure on-prem db with aren't used with Atlas. I can mention here that `--ttl` flag also doesn't matter.

Reply:
That's a good point, and your note below makes --host info clear.

Reply:
We do output this, when we run this command. Can we add a note that if using Atlas, what steps are needed vs providing the steps for self hosted mongo

```
➜  ~ tctl auth sign --format=mongodb --host=mongo --out=mongo
Database credentials have been written to mongo.crt, mongo.cas.

To enable mutual TLS on your MongoDB server, add the following to its
mongod.yaml configuration file:

net:
  tls:
    mode: requireTLS
    certificateKeyFile: /path/to/mongo.crt
    CAFile: /path/to/mongo.cas
```

Reply:
Good call, I'll make a ticket and fix.